prow/jobs/module-manager/module-manager.yaml

```diff
@@ -3,13 +3,13 @@
 
 presubmits: # runs on PRs
   kyma-project/module-manager:
-    - name: pull-lint
+    - name: pull-module-mgr-lint
```

Comment:
is this rename required, since pull-lint job was in `module-manager.yaml`

Reply:
Please have a look at my description
```
- Since prow does not distinguish between Repositories in the job history if they have the same naming
- Example: https://status.build.kyma-project.io/job-history/gs/kyma-prow-logs/pr-logs/directory/pull-lint
This history includes jobs for lifecycle-manager, module-manager, as well as runtime-watcher, because the jobs have the same name
```

I personally prefer if we could have separated PR histories for the repos, to have a better overview in the job history.

Reply:
Thanks for the clarification